.github/workflows/CI.yml

```diff
@@ -43,30 +60,49 @@ jobs:
   make_mason:
     runs-on: ubuntu-latest
     env:
+      # To try to keep this config from running the longest,
+      # use a simpler configuration
+      CHPL_TASKS: fifo
+      CHPL_MEM: cstdlib
+      CHPL_GMP: none
+      CHPL_RE2: bundled
+      CHPL_LLVM: none
+      # check that mason builds
       CHPL_SMOKE_SKIP_MAKE_MASON: false
```

Comment:
Instead of manually setting a quickstart-like configuration, can we do something to tell `util/buildRelease/smokeTest` to source the quickstart script and then if it's testing mason set `CHPL_RE2=bundled`?


Can we switch this to source quickstart and have `util/buildRelease/smokeTest` set `CHPL_RE2=bundled` if mason  is being tested? (I find manually setting the env vars to be tedious and don't want to effectively duplicate quickstart logic)

Reply:
I'll have the CI.yml source the quickstart script first. I'd rather not modify smokeTest right now if I don't have to.

Reply:
Hmm.. or maybe I will modify smokeTest. I'm not so sure I can `export` env vars between run commands here.